bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/creators/ValueCreator.java

```diff
@@ -758,6 +758,9 @@ public static BListInitialValueEntry createListInitialValueEntry(Object value) {
     /**
      * Create a runtime record value with given record type.
      *
+     * <p>To create a record value using a record type defined in Ballerina,
+     * do not use this method and use another createRecordValue method.
+     *
```

Comment:
```suggestion
     * <p>This method creates a record value using a record type defined in Ballerina. 
     * Please use the other @link{ #createRecordValue()} methods instead of this.
     *
```